ids/short_node_id.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ids
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/staking"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/hashing"
+)
+
+const ShortNodeIDLen = ShortIDLen
+
+var (
+	EmptyShortNodeID = ShortNodeID{}
```

Comment:
I've kind of grown to hate this `Empty...` pattern. Feels like we should probably just use the zero value.

Not asking for anything to change in this PR to be clear... Just felt like voicing that.